lib/rideshare_module_methods.rb

```diff
@@ -0,0 +1,58 @@
+# Stores the module methods
+module RideShare
```

Comment:
Nice idea, I think these make more sense as a mix-in but you definitely have the right idea.